mypy.ini

```diff
@@ -41,3 +41,6 @@ ignore_missing_imports = True
 
 [mypy-mordred]
 ignore_missing_imports = True
+
+[mypy-seaborn]
+ignore_missing_imports = True
```

Comment:
stubs are available for sseaborn so we should at least try to sue them and not exclude it here: https://pypi.org/project/types-seaborn/

Reply:
So if I just include that package in the installation, then mypy should be happy? Do I understand that correctly? Will try

Reply:
Problem here: This requires Python>=3.9 , hence the pipeline currently fails as I included this. So we can either 
1) delay this PR until we got rid of Python 3.8 or
2) use the previous solution and have a mini-PR after dropping 3.8.
I'd personally go for 2) since I want this PR merged soon :D

Reply:
I have included it and synced the `types-seaborn` version number with the version number that we use for seaborn. As I am not 100% sure if this is now as intended, please resolve if this is now included properly.

Reply:
please see the way other stubs are imported as additional dependency (eg the pandas types)

im not sure if you can simply use the same versio, they might be synced (ideally) but eg other packages use different versioning

Reply:
I'll move it to a different place and then just use the latest.

Reply:
nothing explicit, espeicially if youve incldued the stubs int he same manner as the other ones and made sure the versions are aligned witht he actual package (which is not soemthing you can assume a priori) then all should be good

Reply:
unless of  course if ti doesnt actually fix the mypy errors (was eg the case for botorch, which is typed but for some reason mypy still says it cant find the info)

Reply:
As discussed, this can be resolved.